basic click info drawer

MaticDiba opened the pull request:
@jeancaffou I have one question regarding this:
```javascript
setTimeout(async () => {
    await this.onMapClick(evt)
}, '100')
```
with this "hack" I was able to achieve opening dialog both on mobile and web versions. If I skip the delay, on mobile, a dialog will be immediately closed. Since I guess the click event is also picked up by dialog backdrop. Did you have any similar issues before? 


Closes #16 
Closes #17

jeancaffou commented:
Hmm in general I try to avoid using `setTimeout` in Vue, and instead use 
```js
this.$nextTick(() => {
  this.onMapClick(evt)
});
```

But in this specific case I would probably take some time to figure out how to stop click propagation. 
Does
```js
mounted () {
  this.mapRef.map.on('click', (evt) => {
    this.onMapClick(evt)
    evt.stopPropagation()
  })
}
```

not work?

MaticDiba commented:
@jeancaffou I agree regarding timeout. I just placed it there to elaborate the problem. But I didn't know about the `$nextTick`.

I tried `stopPropagation` but it does not work since I think that another event is triggered by the backdrop. 

But anyway: I changed the implementation. When testing this on my phone, I found out that having backdrop is making UX quite bad, since user needs to click again to be able to interact with the map. So now I use  `dialog` with `seamless` flag. And I added close button to the dialog itself. I think it provides better UX, and the problem with premature closure is gone :)
